recipe/meta.yaml

```diff
@@ -47,6 +54,7 @@ requirements:
     - importlib_metadata
     - pytorch
     - pytorch =*={{ torch_proc_type }}*
+    - cuda-version =={{ cuda_compiler_version }}  # [cuda_compiler_version not in (undefined, 'None')] 
```

Comment:
I put it in the three because otherwise in another feedstock I saw that the runner tried to bring in two different cuda versions and ran out of storage. The culprit here being that anything compiled for CUDA 11.8 also works on 11.2 and viceversa. Lets try and remove it.